solidity/contracts/BancorNetwork.sol

```diff
@@ -11,6 +12,13 @@ import './utility/interfaces/IWhitelist.sol';
 import './token/interfaces/IEtherToken.sol';
 import './token/interfaces/ISmartToken.sol';
 
+/*
+    Bancor converter dedicated interface
+*/
+contract IBancorConverterExtended is IBancorConverter, IOwned {
```

Comment:
this should be added to the real interface, there might be other use cases for it.